packages/loader/container-loader/src/test/deltaManager.spec.ts

```diff
@@ -216,6 +216,47 @@ describe("Loader", () => {
                     await tickClock(expectedTimeout);
                 });
             });
+
+            describe("Readonly API", () => {
+                it("Should override readonly", async () => {
```

Comment:
Any chance you can add a some test object (data object) in here and assert that it observed proper "readonly" events reaching it?